app/jobs/download_analyses_job.rb

```diff
@@ -39,7 +39,7 @@ def perform(sid, user_id, rt=true)
 
     begin
       @link = if Rails.env.production?
-        "https://#{ENV['HOST'] || ENV['SMTP_DOMAIN']}/zip/#{@filename}"
+        "#{(ENV['URL_USE_SSL'] == 1 || ENV['URL_USE_SSL'] == 'true') && 'https' || 'http'}://#{ENV['URL_HOST'] || ENV['HOST'] || ENV['SMTP_DOMAIN']}/zip/#{@filename}"
```

Comment:
Please use a more readable form:

`protocol = ENV['URL_USE_SSL'].in?([1, '1', 'true']) ? 'https' : 'http'
domain = ENV['URL_HOST'] || ENV['HOST'] || ENV['SMTP_DOMAIN']

"#{protocol}://#{domain}/zip/#{@filename}"`

Reply:
hi @jhmegorei,
 
thx for the tip.
the PR is still in draft and actually some of the used ENV vars will be deprecated.